pysilcam/__main__.py

```diff
@@ -82,7 +85,7 @@ def silcam():
         if args['--nomultiproc']:
             multiProcess = False
         nbImages = args['--nbimages']
-        if (nbImages != None):
+        if (nbImages is not None):
```

Comment:
Here, and below the parathes are not necessary, so I'd personally drop them.
if nbImages is not None: